src/shared/compatibility.js

```diff
@@ -223,11 +228,11 @@ const isIE = /Trident/.test(userAgent);
     // need to be polyfilled for the IMAGE_DECODERS build target.
     return;
   }
-  if (globalScope.Promise && (globalScope.Promise.prototype &&
-                              globalScope.Promise.prototype.finally)) {
+  if (globalThis.Promise && (globalThis.Promise.prototype &&
+                              globalThis.Promise.prototype.finally)) {
```

Comment:
One space of indentation too much here.

Reply:
Whoops, nice catch; although I'm going to blame the search-and-replace feature of my editor for nothing handling that case correctly :-P